Architecting for scale (#5485)
jenkins-infra/jenkins.io: content/doc/book/scaling/architecting-for-scale.adoc

versions of the same tool.

The pre-compiled "Default" option calls whatever is already installed on the

> agent and exists in the machine PATH, but this returns a failure if the tool is not
> installed and its location was not added to the PATH system variable.

Reviewer: ```suggestion
The pre-compiled "Default" option calls whatever is already installed on the agent and exists in the machine PATH. 
This returns a failure if the tool is not installed and its location was not added to the PATH system variable.
```
rearrange sentences and ensure that each sentence is its own line of code (based on full stop .)

resources together with their destruction when they are not needed anymore:

* The plugin:ec2[EC2 Plugin]

>   lets Jenkins use AWS EC2 instances as cloud build resources when it runs out
>   of on-premises agents. The EC2 agents will be dynamically created inside an
>   AWS network and de-provisioned when they are not needed.

Reviewer: ```suggestion
The plugin:ec2[EC2 Plugin] lets Jenkins use AWS EC2 instances as cloud build resources when it runs out of on-premises agents.
The EC2 agents will be dynamically created inside an AWS network and de-provisioned when they are not needed.
```

It is a best practice to take regular backups of your $JENKINS_HOME. A backup
ensures that your Jenkins instance can be restored despite a misconfiguration,
accidental job deletion, or data corruption. See the 
link:https://www.jenkins.io/doc/book/system-administration/backing-up/[Backup policies]
for more details.

=== Finding your $JENKINS_HOME

**Windows**

> If you install Jenkins with the Windows installer, Jenkins is installed as
> a service and the default _$JENKINS_HOME_ will be "C:\Program Files
> (x86)\jenkins".

Reviewer: ```suggestion
If you install Jenkins with the Windows installer, Jenkins is installed as a service and the default _$JENKINS_HOME_ will be "C:\Program Files(x86)\jenkins".
```
combine into one line as they are part of the same sentence
